docs/context-graph/writing.md

```
@@ -0,0 +1,455 @@
+Full flags live in [cli-flow.md](./cli-flow.md); the write loop discipline is taught by the
+`potpie-graph` skill (see [skills.md](./skills.md)).
+
+| Command | Spine / role |
+|---|---|
+| `graph propose --file mutation.json [--ttl 1h]` | Spine A — validate + persist plan (no write) |
+| `graph commit <plan_id> --verify [--approved-by]` | Spine A — apply persisted plan, read-back verify |
+| `graph bulk apply --file <ndjson> [--chunk-size] [--verify]` | Spine A — chunked multi-plan apply |
+| `graph mutate --file … [--dry-run] [--allow-review-required] [--approved-by]` | **legacy wrapper** over propose+commit |
+| `graph mutation-template --kind <…>` | static schema-only skeleton (no host call) |
+| `record --type … --summary …` / MCP `context_record` | Spine B — record→semantic bridge (only MCP write) |
+| `graph history [--entity|--claim|--plan|--mutation|--subgraph]` | committed-write audit trail |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)